### PR TITLE
fix(time): try downloading mirrored data

### PR DESCRIPTION
**Pull request by jrs65**

### CODE REVIEW

File: caput/time.py [hunk 1]
```diff
@@ -1075,11 +1077,22 @@ def timescale(self):
...
+                self.load.download(self.mirror_url + "Leap_Second.dat")
+                self.load.download(self.mirror_url + "finals2000A.all")
+
+                self._timescale = self.load.timescale(builtin=False)
+            except IOError as e:
+                raise IOError(
+                    "Could not get timescale either from an existing installation at "
+                    f"{self.load.directory}, an official download source or the CHIME "
+                    "mirror. Good luck."
+                ) from e
```
Comment (ashill) [inline caput/time.py:1095]: Add a more helpful error message suggesting the user download themselves? Something like "If you have a functioning mirror, try using ctime.skyfield_wrapper.load to manually download the data."

**Review by ashill — APPROVED** [review]

File: caput/time.py [hunk 2]
```diff
@@ -1015,6 +1015,8 @@ class SkyfieldWrapper:
         The JPL ephemeris to use. Defaults to `'de421.bsp'`.
     """
 
+    mirror_url = "https://bao.chimenet.ca/skyfield/"
+
```
[the remark this answers is not in the document]
Comment (jrs65) [reply caput/time.py:1019]: This one is at least configurable in that the user can modify it just be changing the variable on the skyfield_wrapper object (unlike if it was embedded in the function). I'll probably just leave it like this until there's a good use case (or someone else asks) for it to be configurable.

File: caput/time.py (hunk 1 above)
Comment (jrs65) [reply caput/time.py:1095]: Yeah, I can do that.

File: caput/time.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ashill) [reply caput/time.py:1019]: Fair enough.

**Pull request merged**